projects/igniteui-angular/migrations/update-10_2_0/index.ts

```diff
@@ -0,0 +1,45 @@
+import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { UpdateChanges } from '../common/UpdateChanges';
+
+const version = '10.2.0';
+
+export default function (): Rule {
+    return (host: Tree, context: SchematicContext) => {
+        context.logger.info(
+            `Applying migration for Ignite UI for Angular to version ${version}`
+        );
+
+        const update = new UpdateChanges(__dirname, host, context);
+
+        update.addCondition('type_is_invalid', function (
+            matchedOwner: string
+        ): boolean {
+            if (!matchedOwner) {
+                return true;
+            }
+
+            const attrMatch = matchedOwner.match(
+                new RegExp(`type=(["'])(.+?)${'\\1'}`)
+            );
+
+            if (attrMatch?.length > 0) {
+                const attr = attrMatch[0];
+
+                return (
+                    attr === 'type="bootstrap"' ||
```

Comment:
You should be able to check w/ `includes()` rather than specifying both versions of quotes since you've already narrowed down to the property. Still fine like this though